packages/angular-workspace/nimble-angular/src/directives/checkbox/nimble-checkbox-control-value-accessor.directive.ts

```diff
@@ -10,14 +11,30 @@ import { CheckboxControlValueAccessor } from '../../thirdparty/directives/checkb
 @Directive({
     selector:
       'nimble-checkbox[formControlName],nimble-checkbox[formControl],nimble-checkbox[ngModel]',
-    // The following host metadata is duplicated from CheckboxControlValueAccessor
```

Comment:
This doesn't need to necessarily be removed. On Angular upgrades we should be aligning to the Angular CheckboxControlValueAccessor changes / bugfixes